app/services/trainees/create_from_hesa.rb

```diff
@@ -148,7 +136,7 @@ def school_attributes
       end
 
       if hesa_trainee[:employing_school_urn].present?
-        if NOT_APPLICABLE_SCHOOL_URNS.include?(hesa_trainee[:lead_school_urn])
+        if NOT_APPLICABLE_SCHOOL_URNS.include?(hesa_trainee[:employing_school_urn])
```

Comment:
Do we have a data discrepancy?

Reply:
have raise with Claire and Sarah in the ticket